src/node_http2.cc

```diff
@@ -557,11 +557,19 @@ class Http2Session::MemoryAllocatorInfo {
 
     if (mem != nullptr) {
       // Adjust the memory info counter.
+      // TODO(addaleax): Avoid the double bookkeeping we do with
+      // current_nghttp2_memory_ + AdjustAmountOfExternalAllocatedMemory
+      // and provide versions of our memory allocation utilities that take an
+      // Environment*/Isolate* parameter and call the V8 method transparently.
       session->current_nghttp2_memory_ += size - previous_size;
+      session->env()->isolate()->AdjustAmountOfExternalAllocatedMemory(
+          static_cast<int64_t>(size - previous_size));
```

Comment:
Tiny DRY nit: `const size_t new_size = size - previous_size;` and use `new_size` on this and the previous line.

Reply:
Done!